modules/mono/glue/gd_glue.cpp

```diff
@@ -193,10 +193,14 @@ void godot_icall_GD_randomize() {
 	Math::randomize();
 }
 
-double godot_icall_GD_rand_range(double from, double to) {
+double godot_icall_GD_randf_range(double from, double to) {
 	return Math::random(from, to);
 }
 
+int godot_icall_GD_randi_range(int from, int to) {
```

Comment:
Return type and parameters of internal calls should be `int32_t` instead of `int`.

Reply:
Fixed